frontend/src/components/panels/Document.vue

```diff
@@ -332,50 +333,34 @@ export default defineComponent({
 		},
 	},
 	mounted() {
-		registerResponseHandler(ResponseType.UpdateCanvas, (responseData: Response) => {
-			const updateData = responseData as UpdateCanvas;
-			if (updateData) this.viewportSvg = updateData.document;
+		registerJsMessageHandler(UpdateCanvas, (updateData) => {
```

Comment:
Would you mind renaming the arrow function's parameter as the camelCased version of its name name, e.g. `updateCanvas` for this one, throughout?